src/plugins/unified_field_list/public/components/field_stats/field_top_values_bucket.tsx

```diff
@@ -69,7 +88,7 @@ export const FieldTopValuesBucket: React.FC<FieldTopValuesBucketProps> = ({
           >
             {(formattedFieldValue?.length ?? 0) > 0 ? (
               <EuiToolTip content={formattedFieldValue} delay="long">
-                <EuiText size="xs" className="eui-textTruncate" color="subdued">
+                <EuiText size="xs" className="eui-textTruncate" color={color}>
```

Comment:
Let's keep the values color in grey by default rather than in blue (screenshots are from Discover page)

Before:
<img width="393" alt="Screenshot 2022-10-18 at 11 09 46" src="https://user-images.githubusercontent.com/1415710/196388591-3e94674d-869f-42fd-a979-906726547207.png">

In this PR:
<img width="368" alt="Screenshot 2022-10-18 at 11 10 37" src="https://user-images.githubusercontent.com/1415710/196388643-9919006c-ff1b-414a-b2b7-621c7129fb05.png">

Reply:
We could also add a test for `overrideFieldTopValueBar`.

Reply:
Updated here https://github.com/elastic/kibana/pull/143416/commits/54769031be5ac55d822d3ce39b7d844cccf52c89